pytheus/metrics.py

```diff
@@ -15,7 +15,19 @@
 label_name_re = re.compile(r'[a-zA-Z_][a-zA-Z0-9_]*')
 
 
+@dataclass
+class Sample:
+    suffix: str
```

Comment:
the samples have a suffix, depending on how we go creating the Registry, we should pass the metric name once, and for each samples, add the suffixes if they are set

Reply:
What is this suffix thing?

Is it the naming convention of metrics like `_seconds`, `_bytes`, etc?

Reply:
yeah, here is for the count in specific that seems it should end with `_total`

resource: https://prometheus.io/docs/practices/naming/

Reply:
What if a user chooses the metric name `foo_total`? Would we end up with `foo_total_total`? Who decides the responsibility of the naming? I thought it was entirely up to the user

Reply:
it is, but clients seems to simplify it for the user to adapt to the requirements, at least the python one.
If the suffix is already present, it should not be added, this should be done at the registry level.

If we have stronger opinions on this, we can rivisit it while considering the prometheus docs, I'm not completely sure either at this point 🤔

Reply:
I'm not strongly against it, but I found it a bit unclear, so that's why I was asking. So I assume this is just a helper that the client does and prometheus docs are not clear about, so we will try to find if the user has been diligent already not to add it twice, right? In such case, :+1:

Reply:
I've opened #10 
Thanks for raising this point 🙌